rules/xcodeproj.bzl

```diff
@@ -683,8 +683,11 @@ def _xcodeproj_impl(ctx):
         if _is_current_project_file(f)
     ]
 
+    attributes = {}
+    attributes.update(ctx.attr.project_attributes_overrides)
```

Comment:
is there any reason why this can't be passed directly into the `struct` initializer below?